Containerfile

```
@@ -0,0 +1,18 @@
+FROM python:3.9-slim
+
+# Set the working directory
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+# Clone the repository
+RUN git clone https://github.com/LeidenUniversityLibrary/abnormal-hieratic-indexer.git /app
```

The code is in this repository, so we don't need to clone from GitHub :)
```suggestion
COPY . /app
```
Instead of copying everything (including `.github`), we could be more selective in the `COPY` command, or include `.dockerignore`.